data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+	landing
+	invisible
+	source Foundry
+	to offer
+		has "Hai Leaks Response 2: active"
+	on offer
+		log "People" "Terry Adrianopoulos" `Protege to the Syndicate CEO, Terry Adrianopoulos has been nominated by her mentor Alastair Korban to represent the Syndicate's interests in the crisis of the Hai's secrecy. She is a very capable woman who appears well equipped for climbing the corporate ladder.`
+		conversation
+			`Approaching Foundry, you are hailed on a private encrypted channel and directed to a private hangar where you are greeted personally by Syndicate CEO Alastair Korban.`
+			`	He shakes your hand and smiles warmly. "Apologies, Captain <last>, do excuse the cloak and dagger, but in the current media climate you and I are both too well-known to be seen meeting without causing the rumor mill to grind out a new story."`
+			`	"Now, allow me to introduce you to my protege, Terry Adrianopolous. She will be accompanying you as the Syndicate representative."`
+			`	"Pleasure to meet you," she says with a broad smile. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She is pleasant and approachable, and very articulate, but from your brief glimpse beforehand you have no doubt she's cut from the sort of cloth that makes her future CEO material. You are surprised that a woman would have made it this far in the Syndicate, but if she has gotten this far, she must be very skilled at what she does.`
+			`	"By the way," says Alastair, pausing briefly as he turns to leave, "The Navy and my security forces are working jointly to get some of our message relays up in the Far North to link through Ultima Thule. The pirates are wont to destroy them whenever they find them, but we should be able to rely on at least basic communication to get through if you need it, rather than flying back and forth for literally every question. That's all."`
+			branch delayedretire
+				not "free worlds reconciliation"
+			`	He leans in close to you and says quietly, "I know I said I was retiring, but Terry convinced me to clean up my mess first. I am atoning for my failures. Do look after her; she's a very loyal colleague."`
+			label delayedretire
+			`	Alastair says goodbye and departs, leaving you with Terry.`
+				decline
+
+#This gets picked up silently on Hai-home when landing for response 2 - leads to go kill pirates mission with chat on Allhome.
+mission "Hai Leaks Response 3"
+	name "Meet with Security Forces on Allhome"
+	description "Meet with the Security Forces on <destination> to fight pirates."
+	landing
+	source "Hai-home"
+	destination "Allhome"
+	to offer
+		has "Hai Leaks Response 2: done"
+
+#Go kill pirates mission with chat on Allhome.
+mission "Hai Leaks Response 4"
+	name "Defeat the Black Lich"
+	description "Defeat Black Lich's Marauder Leviathan at <waypoints>, then return to <origin>."
+	autosave
+	landing
+	source "Allhome"
+	waypoint "Waypoint"
+	destination "Allhome"
```

Comment:
> It's necessary if you also have a waypoint.

You sure about that?

Reply:
Well, at an earlier stage in the process it didn't work without it. I do not recall the exact circumstances of that testing at the time - it being nearly 2 years ago.